frontend/vue.config.js

```diff
@@ -17,9 +17,9 @@ https://v4.webpack.js.org/configuration/dev-server/
 */
 
 module.exports = {
-    publicPath: process.env.NODE_ENV === 'production' ? '/static/dist/' : 'http://127.0.0.1:8080',
-    outputDir: '../static/dist',
-    indexPath: '../../templates/index.html',
+    publicPath: process.env.NODE_ENV === 'production' ? '/static/' : 'http://127.0.0.1:8080',
+    outputDir: '../static',
```

Comment:
changed public path and output directory from static/dist/ to static/ for the deployment purpose.